migrator/migrations/m_180_to_m_181_move_to_blobstore/schema/convert_blobs_test.go

```diff
@@ -16,5 +17,5 @@ func TestBlobSerialization(t *testing.T) {
 	assert.NoError(t, err)
 	conv, err := ConvertBlobToProto(m)
 	assert.NoError(t, err)
-	assert.Equal(t, obj, conv)
+	assert.True(t, proto.Equal(obj, conv))
```

Comment:
Would it make sense in a follow-up to have a function that dumps the message diff (at least for the message fields that do not match) ? It would likely make the debugging easier. This can be done in a follow-up ticket.

Reply:
@rhybrillou I agree with you on this, and this is also something @janisz mentioned. We could implement Equal assertion for proto messages and use it. It would be great if we could do that before the actual V2 switch.